sources/packages/web/src/services/AppConfigService.ts

```diff
@@ -101,6 +101,7 @@ export class AppConfigService {
     type: ClientIdType,
     authService?: KeyCloak.KeycloakInstance,
     isBasicBCeID?: boolean,
+    isDisabled?: boolean
```

Comment:
Please consider renaming to something that indicates that the user is disabled, like "isUserDisabled".